awx/main/analytics/collectors.py

```diff
@@ -290,12 +298,16 @@ def _copy_table(table, query, path):
                                  FROM main_unifiedjob
                                  JOIN django_content_type ON main_unifiedjob.polymorphic_ctype_id = django_content_type.id
                                  LEFT JOIN main_organization ON main_organization.id = main_unifiedjob.organization_id
-                                 WHERE (main_unifiedjob.created > {0} OR main_unifiedjob.finished > {0})
+                                 WHERE ((main_unifiedjob.created > {0} AND main_unifiedjob.created < {1})
```

Comment:
we should probably do `(main_unifiedjob.created > {0} AND main_unifiedjob.created <= {1})` the the `<=` less or equal

Although the probability we'll hit this in prod is near 0.

We might hit this in a perf test env, where we'll probably have mock up data aligned to full seconds, not microseconds

Reply:
Done